.github/workflows/bump-chrome-version.yml

```
@@ -0,0 +1,65 @@
+name: Update Chrome Version used for wasm testing
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Branch
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email github-actions[bot]@users.noreply.github.com
+          git checkout -b update-chrome-version-${{ github.run_id }}
+
+      - name: Run UpdateToLatestVersion
+        run: >-
+          make -C src/mono/wasm build-tasks &&
+          .dotnet/dotnet build src/mono/wasm/bump_chrome_version.proj -p:Configuration=Release &&
```

```suggestion
          .dotnet/dotnet build eng/testing/bump-chrome-version.proj -p:Configuration=Release &&
```
There is no file `src/mono/wasm/bump_chrome_version.proj`